static/src/styles/core/color.styl

```diff
@@ -0,0 +1,7 @@
+$Color__light = #F2F2F2;
```

Comment:
Nothing hard and fast, just naming the prefix after the file it's contained in, maybe this file should be called `Color` instead of `color` for consistency with other components.

As I said, BEM doesn't really have anything to say about these variables. I've conjured this idea that `Color` is a block, and `light`, `bright`, etc are all elements of it.